style.scss

```diff
@@ -0,0 +1,1183 @@
+              left: -54px;
+              top: 14px;
+              position: absolute;
+              height: 1px;
+              width: 46px;
+              transition: all 0.3s ease 0s;
+              background-color: #22BF95;
+            }
+          }
+        }
+      }
+
+      .arrow1 {
+        cursor: pointer;
+        position: absolute;
+        top: 24px;
+        right: 24px;
+      }
+
+      .arrow2 {
+        cursor: pointer;
+        position: absolute;
+        top: 56px;
+        right: 24px;
+      }
+    }
+
+    .slid {
+      h3 {
+        @include fontProp(32px, 40px, #1A1A1A);
+        font-family: "Actor", sans-serif;
+        letter-spacing: 1.5px;
+        text-transform: uppercase;
+        padding: 32px 0 24px 16px;
+      }
+
+      p {
+        @include fontProp(16px, 24px, #191919);
+        padding-left: 16px;
+        border-left: 1.5px solid #22BF95;
+      }
+    }
+  }
+
+}
+
+// -----FAQs------------
+
+.faqs {
+  padding: 24px 20px;
+
+  .title {
+    margin-bottom: 24px;
+
+    hr {
+      margin-bottom: 10px;
+      border: none;
+      height: 1.5px;
+      background-color: #B2B2B2;
+    }
+
+    img {
+      width: 110px;
+      height: 18px;
+    }
+  }
+
+  .block {
+    span {
+      font-family: "Actor", sans-serif;
+      @include fontProp(40px, 48px, #E5E5E5);
+      text-align: center;
+      letter-spacing: 2px;
+      text-transform: uppercase;
+      margin-right: 35px;
+    }
+
+    .question {
+      display: flex;
+      align-items: center;
+      height: 114px;
+      cursor: pointer;
+
+      p {
+        flex-grow: 1;
+        font-family: "Aclonica", sans-serif;
+        @include fontProp(16px, 24px, #191919);
+        overflow-y: visible;
+      }
+
+      > div {
+        margin-left: 78px;
+        width: 16px;
+
+      }
+
+    }
+
+    .answer{
+      display: none;
+    }
+
+    &.blockActive {
+      .question{
+        >div{
+          transform: scale(1, -1);
+        }
+      }
+      .answer {
+        display: flex;
+        align-items: center;
+        height: 90px;
+
+        p {
+          flex-grow: 1;
+          font-family: "Aclonica", sans-serif;
+          @include fontProp(14px, 24px, #191919);
+          overflow-y: visible;
+        }
+
+        > div {
+          margin-left: 7px;
+          width: 16px;
+        }
+      }
+    }
+  }
+
+  > :last-child {
+    text-align: center;
+    margin-top: 20px;
+
+    a {
+      font-family: "Acme", sans-serif;
+      text-decoration: none;
+      text-transform: uppercase;
+      @include fontProp(12px, 24px, #22BF95);
+      letter-spacing: 2px;
+
+      &:hover {
+        @include fontProp(14px, 24px, #01F973);
+      }
+    }
+  }
+
+}
+
+
+// -----FOOTER--------------
+
+footer {
+  background-color: #F2F2F2;
+  font-family: "Aclonica", sans-serif;
+  font-weight: 400;
+
+  .wrapper {
+    div {
+      nav {
+        font-family: "Acme", sans-serif;
+        padding: 17px;
+
+        ul {
```

Comment:
![image](https://user-images.githubusercontent.com/84564899/120075176-e2f9ea00-c0a8-11eb-9bd0-993c9818dccf.png)
link text should be inside its container